appium/tests/screenobjects/email.screen.ts

```diff
@@ -52,7 +52,7 @@ class EmailScreen extends BaseScreen {
   }
 
   checkEmailAddress = async (email: string) => {
-    const selector = `~${email}`;
+    const selector = `-ios class chain:**/XCUIElementTypeStaticText[\`label == "${email}"\`]`;
```

Comment:
Whenever you see yourself using this kind of complicated selector, please add accessibility identifier instead, and query by that.

Reply:
will try